src/root.component.tsx

```diff
@@ -1,18 +1,44 @@
 import React from "react";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import { defineConfigSchema, validators } from "@openmrs/esm-module-config";
+import Navbar from "./navbar/navbar.component";
 
-defineConfigSchema("pih-esm-refapp-navbar", {
-  displayGreeting: {
-    validators: [validators.isBoolean],
-    default: true
+defineConfigSchema("@pih/esm-refapp-navbar", {
+  links: {
+    home: {
+      url: {
+        default: (window as any).openmrsBase
```

Comment:
"as any" is a TypeScript thing since it's using the existing window object from the browser but that's never explicitly defined anywhere?

Reply:
Pretty much. Typescript does have type information for "window", but it only has the stuff that "window" normally has, like "location". Typescript simply doesn't know about custom stuff tacked onto "window", such as "openmrsBase".